google/cloud/forseti/services/inventory/base/resources.py

```diff
@@ -42,7 +43,9 @@ def size_t_hash(key):
     Returns:
         str: The hashed key.
     """
-    return '%u' % ctypes.c_size_t(hash(key)).value
+    # pylint: disable=no-member
+    hash_digest = hashlib.blake2b(key.encode()).hexdigest()
```

Comment:
Can you put the pylint disable comment beside the line if the length is allowed:

`hash_digest = hashlib.blake2b(key.encode()).hexdigest()  # pylint: disable=no-member`

if not, would be good if you can enable it afterward

```
    # pylint: disable=no-member
    hash_digest = hashlib.blake2b(key.encode()).hexdigest()
    # pylint: enable=no-member
```